docs/api/apiv3/paths/work_schedule_non_working_days.yml

```diff
@@ -0,0 +1,106 @@
+---
+get:
+  summary: Lists non-working days
+  operationId: list_non_working_days
+  tags:
+    - Work Schedule
+    - Non working days
+  description: |-
+    Lists non-working days for a given date interval.
+  parameters:
+    - name: include_weekend
+      in: query
+      description: |-
+        Include weekend days in the response when `true`.
+      required: false
+      schema:
+        default: false
+        type: boolean
+    - name: filters
+      in: query
+      description: |-
+        JSON specifying filter conditions.
+
+        Accepts the same format as returned by the [queries](https://www.openproject.org/docs/api/endpoints/queries/)
+        endpoint. Currently supported filters are:
+
+        + interval: The inclusive date interval to scope non working days to look up.
+
+          Example: `{ "interval": { "operator": "<>d", "values": ["2022-05-02","2022-05-26"] } }` would return non working days
+          between May 5 and May 26 2022, inclusive.
+      example: '[{ "interval": { "operator": "<>d", "values": ["2022-05-02","2022-05-26"] } }]'
+      required: false
```

Comment:
`filter` is optional query param, but how does this endpoint behaves if the filter is not provided? It should go to the description, which currently only says, that non-working days for an interval are returned.

Reply:
Yep good point. That's not specified for now.

Reply:
It has been added. Thanks.